scenes/src/app/scene-viewer/scene-viewer.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Component,
+  ComponentFactoryResolver,
+  Input,
+  NgModule,
+  OnInit,
+  ViewChild,
+  ViewContainerRef,
+  ViewEncapsulation
+} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {InputSceneModule} from '../scenes/input/input-scene';
+import {ButtonSceneModule} from '../scenes/button/button-scene';
+import {ActivatedRoute} from '@angular/router';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'app-scene-viewer',
+  templateUrl: './scene-viewer.html',
+  styleUrls: ['./scene-viewer.scss']
+})
+export class SceneViewer implements OnInit {
+  @Input() hueRotation: string;
```

Comment:
Can you comment how this input works so that people who are making scenes in the future know how to use it